index.js

```diff
@@ -406,3 +406,13 @@ export const clickable = {
   clickableNotToggle: 'inset-0 absolute',
   label: `px-12 ${label.label} py-8! cursor-pointer`, 
 }
+
+export const ccCombobox = {
```

Comment:
only "combobox" here, right? Without the "cc"?
```suggestion
export const combobox = {
```

Reply:
Great catch, damn copy paste 😁